tools/raspicam_view.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python
```

Comment:
Also this tool needs a better name like `imv_view`

Reply:
Renamed; I originally didn't want to link the name to motion vectors because the tool can be also used to show only the image -- but there are already standard tools for that which are part of the ROS ecosystem.